### ISSUE TITLE
Exception triggered when recipients download their own files

### ISSUE THREAD

**Issue by aetdr** [globaleaks/GlobaLeaks#3220]
Hi @evilaliv3 

After upgrading to v 4.9.4, admins get this error by mail:

```
Platform: host
Host: host.domain.name (***.onion)
Version: 4.9.4

URL: /status/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx

User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36

Error Message: TypeError: Cannot read properties of undefined (reading 'finally')

Stacktrace:
[
  {
    "columnNumber": 620923,
    "lineNumber": 64,
    "fileName": "https://host.domain.name/js/scripts.min.js",
    "functionName": "d.e.downloadWBFile",
    "source": "    at d.e.downloadWBFile (https://host.domain.name/js/scripts.min.js:64:620923)"
  },
  {
    "columnNumber": 114534,
    "lineNumber": 8,
    "fileName": "https://host.domain.name/js/scripts.min.js",
    "source": "    at https://host.domain.name/js/scripts.min.js:8:114534"
  },
  {
    "columnNumber": 151847,
    "lineNumber": 8,
    "fileName": "https://host.domain.name/js/scripts.min.js",
    "functionName": "i",
    "source": "    at i (https://host.domain.name/js/scripts.min.js:8:151847)"
  },
  {
    "columnNumber": 102807,
    "lineNumber": 8,
    "fileName": "https://host.domain.name/js/scripts.min.js",
    "functionName": "d.$eval",
    "source": "    at d.$eval (https://host.domain.name/js/scripts.min.js:8:102807)"
  },
  {
    "columnNumber": 103052,
    "lineNumber": 8,
    "fileName": "https://host.domain.name/js/scripts.min.js",
    "functionName": "d.$apply",
    "source": "    at d.$apply (https://host.domain.name/js/scripts.min.js:8:103052)"
  },
  {
    "columnNumber": 151932,
    "lineNumber": 8,
    "fileName": "https://host.domain.name/js/scripts.min.js",
    "functionName": "HTMLSpanElement.<anonymous>",
    "source": "    at HTMLSpanElement.<anonymous> (https://host.domain.name/js/scripts.min.js:8:151932)"
  },
  {
    "columnNumber": 17501,
    "lineNumber": 8,
    "fileName": "https://host.domain.name/js/scripts.min.js",
    "functionName": "Ut",
    "source": "    at Ut (https://host.domain.name/js/scripts.min.js:8:17501)"
  },
  {
    "columnNumber": 20371,
    "lineNumber": 8,
    "fileName": "https://host.domain.name/js/scripts.min.js",
    "functionName": "HTMLSpanElement.n",
    "source": "    at HTMLSpanElement.n (https://host.domain.name/js/scripts.min.js:8:20371)"
  }
]

```

**Comment by evilaliv3** [comment]
Thank you for reporting this @aetdr 

This is a minor bug triggered when a recipient download the file; While the download happens correctly, at the end of it the code trigger an exception notification.

**Comment by evilaliv3** [comment]
Solved in 4.9.5
